pandas/_libs/tslibs/period.pyx

```diff
@@ -73,7 +73,7 @@ from pandas._libs.tslibs.tzconversion cimport tz_convert_utc_to_tzlocal
 
 cdef:
     enum:
-        INT32_MIN = -2_147_483_648
+        INT32_MIN = -2_147_483_648LL
```

Comment:
```suggestion
        INT32_MIN = -2_147_483_648L
```

Rather pedantic but the size of a long is at least 32 bits, so no need for long long here.

Reply:
FYI here's some interesting info on why we probably need this suffix depending on the architecture / compilation standard:

https://stackoverflow.com/questions/9941261/warning-this-decimal-constant-is-unsigned-only-in-iso-c90#20910712

Reply:
So ignore above comment; I think LL might be correct here